awx/ui_next/src/components/ResourceAccessList/__snapshots__/ResourceAccessListItem.test.jsx.snap

```diff
@@ -840,6 +840,7 @@ exports[`<ResourceAccessListItem /> initially renders succesfully 1`] = `
                                                         isClosable={false}
                                                         numChips={5}
                                                         onClick={[Function]}
+                                                        onOverflowChipClick={[Function]}
```

Comment:
May not be related to the (now resolved) zuul failures. Nevertheless, `jest` thinks these snapshots should be updated so I did it.